pkg/model/stack_serializer_test.go

```diff
@@ -584,3 +584,108 @@ func Test_restartFile(t *testing.T) {
 		})
 	}
 }
+
+func Test_endpoints(t *testing.T) {
+	tests := []struct {
+		name     string
+		manifest []byte
+		expected EndpointSpec
+	}{
+		{
+			name: "rule with name",
+			manifest: []byte(`name: test
+services:
+  app:
+    ports:
+      - 9213
+    image: okteto/vote:1
+endpoints:
+  app:
+    - path: /
+      service: app
+      port: 9213`),
+			expected: EndpointSpec{
+				"app": Endpoint{
+					Annotations: make(Annotations),
+					Labels:      make(Labels),
+					Rules: []EndpointRule{
+						{
+							Service: "app",
+							Path:    "/",
+							Port:    9213,
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "rule with name and annotations/labels",
+			manifest: []byte(`name: test
+services:
+  app:
+    ports:
+    - 9213
+    image: okteto/vote:1
+endpoints:
+  app:
+    annotations:
+      key: value
+    labels:
+      key: value
+    rules:
+    - path: /
+      service: app
+      port: 9213`),
+			expected: EndpointSpec{
+				"app": Endpoint{
+					Annotations: Annotations{"key": "value"},
+					Labels:      Labels{"key": "value"},
+					Rules: []EndpointRule{
+						{
+							Service: "app",
+							Path:    "/",
+							Port:    9213,
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "direct rules",
+			manifest: []byte(`name: test
+services:
+  app:
+    ports:
+    - 9213
+    image: okteto/vote:1
+endpoints:
+  - path: /
+    service: app
```

Comment:
for the simple mode, do we need to specify the service too? since there's only one, maybe we can assume it?

Reply:
I think that if you just want to create an endpoint for a stack with one service, setting the service to public will be the easiest way to have access to that.

Reply:
@jLopezbarb I agree